composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -463,6 +463,15 @@ def initial_platform
           platform
         end
 
+        def clean_dependency_url(dependency_url)
```

Comment:
If we don't do this, the URLs returned look like: `https://x-a...ken:@github.com/dependabot-fixtures/this-repo-does-not-exist.git` including the truncated `x-access-token` part, which I think looks pretty gnarly

Reply:
Yep, saw this in a composer test and was really confused
![Screen Shot 2022-01-11 at 4 04 33 PM](https://user-images.githubusercontent.com/12107187/149021263-217eb4ae-765b-4836-9e20-6ca09c47a572.png)

Looks like the tests are passing when urls are passed to `clean_dependency_url` to truncate the x-access token though 👍🏾